core/arch/arm/plat-sam/sama7g5.h

```diff
@@ -0,0 +1,404 @@
+/* SPDX-License-Identifier: BSD-Source-Code */
```

Comment:
This SPDX doesn't match with the license below. Which license do you intend to use?
If it's all the same to you, I'd recommend `BSD-2-Clause` since that's what we're mostly using here.

Reply:
Updated to `BSD-2-Clause`.